### PR TITLE
updated tutorial

### PR DESCRIPTION
**Pull request by alixdamman**

### CODE REVIEW

**Comment by alixdamman** [comment]
To run the notebook when building the doc, we need to save it without output:
https://nbsphinx.readthedocs.io/en/0.2.13/executing-notebooks.html

File: doc/source/changes/version_0_23.rst.inc [hunk 1]
```diff
@@ -110,6 +110,11 @@
     >>> wb[0]
     <larray.io.excel.Sheet [Book1]Sheet1>
 
+* path to the directory continaing example files is given the variable `EXAMPLE_FILES_DIR`:
```
Comment (gdementen) [inline doc/source/changes/version_0_23.rst.inc]: * containing
* given by

File: doc/source/changes/version_0_24.rst.inc [hunk 2]
```diff
@@ -0,0 +1,18 @@
+New features
```
Comment (gdementen) [inline doc/source/changes/version_0_24.rst.inc]: kill the file for now

File: doc/source/changes/version_0_23.rst.inc [hunk 3]
```diff
@@ -110,6 +110,11 @@
...
+
+    >>> from larray import *
+    >>> pop = read_csv(EXAMPLE_FILES_DIR + 'pop.csv')
```
Comment (gdementen) [inline doc/source/changes/version_0_23.rst.inc]: I wonder if EXAMPLE_DATA_DIR wouldn't be slightly better? (to be consistent with load_example_data)

**Pull request merged**